components/chainhook-cli/src/scan/stacks.rs

```diff
@@ -95,6 +99,7 @@ pub async fn get_canonical_fork_from_tsv(
         })
         .expect("unable to spawn thread");
 
+    let stacks_db = open_readonly_stacks_db_conn(&config.expected_cache_path(), ctx).unwrap();
```

Comment:
Instead of unwrapping, could we wait. loop and retry? opening could be temporarily impossible because another thread is writing, and that should be something that we're tolerating.